### DISCUSSION TITLE
view.genial.ly => Malware detection

### DISCUSSION THREAD

**Discussion by GRUXXKi** [AmauriC/tarteaucitron.js#1145] [General]
Bonjour,

J'ai installé tarteaucitron sur le site d'un de mes clients. Récemment nous avons reçu une alerte de Google Ads comme quoi le site de mon client était un logiciel malveillant avec désactivation de la publicité.

Le message de google me pointe alors des liens avec ni queue ni tête de site d'arnaques...

Je fais une première recherche, pas moyen de trouver la moindre chose.

Je fais alors un test avec le site quttera.com qui me trouve une erreur. Mais pour en savoir plus, abonnement obligatoire. Je prends...

Le test me fait un rapport détaillé et me sort :
view.genial.ly => GenericMaliciousObject

Et la je trouve ce fichier dans ma copie de tarteaucitron.service.js ( je n'ai pas eu le choix, impossible de le faire fonctionner en CDN).

Pour info : "version": 20230203,

J'ai vérifié dans le CDN actuel https://cdnjs.cloudflare.com/ajax/libs/tarteaucitronjs/1.14.0/tarteaucitron.services.min.js il y a bien également référence ) ce service.

A priori ça n'a pas l'air bien dangereux pourtant...

D'autres personnes ont déjà eu ce genre de retour ?

**Comment by AmauriC** [comment]
Bonjour,

Genial.ly est un service en ligne pour charger des pdf tout à fait légitime, ce n'est pas ce service, ni tarteaucitron qui ajoutent des liens vers des sites d'arnaques sur votre site, ce n'est pas la bonne piste ;)

**Comment by GRUXXKi** [comment]
Oui, aucun doute que le site soit légitime.

Pour autant j'ai supprimé dans le fichier /tarteaucitron.services.min.js l'ajout du service et le site est repassé de nouveau à sécurisé.

**Comment by AmauriC** [comment]
Étrange, le service a été ajouté il y a plus de 2 ans et je n'ai jamais eu ce retour: https://github.com/AmauriC/tarteaucitron.js/commit/7ad4c0aad028546ad3869fd593794c6c58327e91
Et surtout ça n'explique pas l'ajout des liens sur votre site, impossible que ça vienne du fichier tarteaucitron.

**Reply by GRUXXKi** [reply]
En fait les liens ne sont jamais apparu sur le site du client. Je pense que c'est un chaînage du site de mon client vers genial.ly qui a peut être eu un problème temporaire de sécurité ? Le service de sécurité crawl tous les liens en descendant jusqu'à un site potentiellement "dangereux"

Sinon pour info, les rapports me retourne encore ça :+1: 

File name / page address www.SITEDUCLIENT.fr/js/tarteaucitron/tarteaucitron.services.js
Type ASCII
Severity Potentially Suspicious
Name PS.Hidden.Elem.gen
Detected by js-emulation-layer
Page MD5 C30FACBA7BB67E2D9F84A7E7859563F6
Details Detected procedure that is commonly used in suspicious activity.
